modules/account-lib/package.json

```diff
@@ -35,7 +35,7 @@
   "dependencies": {
     "@bitgo/blake2b": "^3.0.0",
     "@bitgo/bls": "^0.1.0",
-    "@bitgo/statics": "^6.12.0-rc.2",
+    "@bitgo/statics": "^6.12.0-rc.3",
```

Comment:
please do not modify the statics version - this should only be changed during the release process.